media/plg_system_tour/js/tours.js

```diff
@@ -0,0 +1,55 @@
+(function ($) {
+    $(document).ready(function() {
```

Comment:
Because I told him not to and tether has been abandoned which basically means shepherd is a non starter.